podvm/Makefile

```diff
@@ -17,6 +17,7 @@ KATA_AGENT_SRC := ../../kata-containers/src/agent
 STATIC_LIBSECCOMP_BUILDER := ../../kata-containers/ci/install_libseccomp.sh
 AGENT_PROTOCOL_FORWARDER_SRC := ../
 
+ifeq ($(CLOUD_PROVIDER),libvirt)
```

Comment:
We are creating qcow2 for all the providers in addition to native images (eg. AMIs for AWS).

Reply:
How about I just create another target, eg. "make template" which is valid only for vsphere and remove the conditional on the "image" target ?

Reply:
Can you simply have `ifeq ($(CLOUD_PROVIDER),vsphere)` and the specifics which are applicable only for vsphere ?

Reply:
OK I'm confused. Perhaps I don't understand the objective.
 
Before these are the instructions in the README under vsphere for making a template

cd vsphere/image && make
```
For a rhel podvm image, try:
```
cd vsphere/image && PODVM_DISTRO=rhel make

Using the word image is confusing as it is used to build the CAA container image and pushes it to quay.

How do I distinguish between wanting a ubuntu template vs a rhel template?
Where do I put the RHEL8.iso I will be using to generate the template?

Reply:
Making sure I understand this - The default $(IMAGE_FILE) target isn't applicable for vsphere. For example, we don't create a cloudinit image with cloud-localds and we don't really generate an image file. So, to be able to do CLOUD_PROVIDER=vsphere make image, we have to specify a different "image" target. But then, if I understood your comment above correctly,  we can't create a vsphere specific qcow2 anymore.  So, the way to make it work is define a new target - template, which does image generation the vmware way and keep image as is so that `CLOUD_PROVIDER=vsphere make image`  provides a qcow2 and `CLOUD_PROVIDER=vsphere make template` creates a vcenter template.